### PR TITLE
Feat: Adds labelLeft props to TextField

### PR DESCRIPTION
**Pull request by anacolell**
New props to be able to position the labels on the left of the textField instead of on top.
Screenshot 2022-06-24 at 5 00 47 PM

### CODE REVIEW

File: src/components/inputs/textField/textField.component.tsx [hunk 1]
```diff
@@ -112,74 +117,150 @@ export const TextField: FC<TextFieldProps> = React.forwardRef(
...
             sx={{
-              color: 'common.black',
-              '&.Mui-focused': {
-                color: 'common.black',
-              },
+              display: 'flex',
+              alignItems: 'center',
+              justifyContent: 'flex-end',
             }}
           >
```
Comment (ksiabani) [inline src/components/inputs/textField/textField.component.tsx:133]: Please don't use `Box` and `sx`. Use a `div` and some class instead. Or am I missing something?

File: src/components/inputs/textField/textField.component.tsx [hunk 2]
```diff
@@ -112,74 +117,150 @@ export const TextField: FC<TextFieldProps> = React.forwardRef(
...
+              className={`${styles.label} ${styles.labelLeft}`}
+              htmlFor={id}
+              disableAnimation
+              required={required}
+              sx={{
+                color: 'common.black',
+                '&.Mui-focused': {
+                  color: 'common.black',
+                },
+              }}
```
Comment (ksiabani) [inline src/components/inputs/textField/textField.component.tsx:145]: I know this has been here before, but can you now replace it with CSS variables? I know that there are others below, but let's at least, fix one at a time. Ignore the rest ;)

Thanks

**Review by ksiabani — COMMENTED** [review]
The UI looks good, I have let two comments ;)

File: src/components/inputs/textField/textField.component.tsx (hunk 1 above)
Comment (anacolell) [reply src/components/inputs/textField/textField.component.tsx]: Changed it to a div!

File: src/components/inputs/textField/textField.component.tsx (hunk 2 above)
Comment (anacolell) [reply src/components/inputs/textField/textField.component.tsx]: Done :)

**Pull request merged**